test/test-sourcemapper.ts

```diff
@@ -52,16 +54,37 @@ function testTool(
   const outputFilePath = path.join(BASE_PATH, relativeOutputFilePath);
 
   describe('sourcemapper for tool ' + tool, () => {
+    const logger = new MockLogger();
     let sourcemapper: sm.SourceMapper;
 
-    it('for tool ' + tool, async () => {
-      const start = Date.now();
-      sourcemapper = await sm.create([mapFilePath]);
-      assert(
-        Date.now() - start < QUICK_MILLISECONDS,
-        'should create the SourceMapper quickly'
-      );
-    });
+    it(
+      'for tool ' + tool + ' sourcemapper should be created correctly',
+      async () => {
+        const start = Date.now();
+        sourcemapper = await sm.create([mapFilePath], logger);
+        assert(
+          Date.now() - start < QUICK_MILLISECONDS,
+          'should create the SourceMapper quickly'
+        );
+
+        // Verify if the debugging information is correctly printed.
+        assert.notStrictEqual(
```

Comment:
Could be just me, but I find the notStrictEqual(...indexOf(...), -1) to be hard to read.  Maybe matches instead?

Reply:
Done.